From 7b61f9ebf6155adab7b46d13ffc26989de1128ff Mon Sep 17 00:00:00 2001
From: Raphael Mounier <mounierr07@gmail.com>
Date: Sun, 5 Mar 2023 15:05:51 +0100
Subject: [PATCH] Remove huawei hardware fingerprint

Remove huawei hardware fingerprint provide by phh vendor_interfaces repos
---
 .../biometrics/fingerprint/2.1/Android.bp       | 16 ----------------
 .../2.1/IExtBiometricsFingerprint.hal           | 17 -----------------
 .../2.1/IFidoAuthenticationCallback.hal         |  5 -----
 3 files changed, 38 deletions(-)
 delete mode 100644 vendor/huawei/hardware/biometrics/fingerprint/2.1/Android.bp
 delete mode 100644 vendor/huawei/hardware/biometrics/fingerprint/2.1/IExtBiometricsFingerprint.hal
 delete mode 100644 vendor/huawei/hardware/biometrics/fingerprint/2.1/IFidoAuthenticationCallback.hal

diff --git a/vendor/huawei/hardware/biometrics/fingerprint/2.1/Android.bp b/vendor/huawei/hardware/biometrics/fingerprint/2.1/Android.bp
deleted file mode 100644
index 572f17a..0000000
--- a/vendor/huawei/hardware/biometrics/fingerprint/2.1/Android.bp
+++ /dev/null
@@ -1,16 +0,0 @@
-// This file is autogenerated by hidl-gen -Landroidbp.
-
-hidl_interface {
-    name: "vendor.huawei.hardware.biometrics.fingerprint@2.1",
-    root: "vendor",
-    srcs: [
-        "IExtBiometricsFingerprint.hal",
-        "IFidoAuthenticationCallback.hal",
-    ],
-    interfaces: [
-        "android.hardware.biometrics.fingerprint@2.1",
-        "android.hidl.base@1.0",
-    ],
-    gen_java: true,
-}
-
diff --git a/vendor/huawei/hardware/biometrics/fingerprint/2.1/IExtBiometricsFingerprint.hal b/vendor/huawei/hardware/biometrics/fingerprint/2.1/IExtBiometricsFingerprint.hal
deleted file mode 100644
index 453be4f..0000000
--- a/vendor/huawei/hardware/biometrics/fingerprint/2.1/IExtBiometricsFingerprint.hal
+++ /dev/null
@@ -1,17 +0,0 @@
-package vendor.huawei.hardware.biometrics.fingerprint@2.1;
-
-import android.hardware.biometrics.fingerprint@2.1::IBiometricsFingerprint;
-import vendor.huawei.hardware.biometrics.fingerprint@2.1::IFidoAuthenticationCallback;
-
-interface IExtBiometricsFingerprint extends android.hardware.biometrics.fingerprint@2.1::IBiometricsFingerprint {
-	setLivenessSwitch(int32_t needLivenessAuthentication) generates(int32_t debugErrno);
-	checkNeedReEnrollFinger() generates (int32_t debugErrno);
-	removeUserData(int32_t gid, string storePath) generates (int32_t debugErrno);
-	updateSecurityId(int64_t security_id) generates (int32_t debugErrno);
-	getFpOldData() generates (int32_t fingerId);
-	verifyUser(IFidoAuthenticationCallback fidoClientCallback, int32_t groupId, string aaid, vec<int8_t> nonce) generates (int32_t debugErrno);
-	getTokenLen() generates (int32_t tokenLen);
-	setCalibrateMode(int32_t do_sensor_calibration) generates (int32_t setModeStatus);
-	checkNeedCalibrateFingerprint() generates (int32_t sensor_calibration_status);
-	sendCmdToHal(int32_t cmdId) generates (int32_t result);
-};
diff --git a/vendor/huawei/hardware/biometrics/fingerprint/2.1/IFidoAuthenticationCallback.hal b/vendor/huawei/hardware/biometrics/fingerprint/2.1/IFidoAuthenticationCallback.hal
deleted file mode 100644
index 040208e..0000000
--- a/vendor/huawei/hardware/biometrics/fingerprint/2.1/IFidoAuthenticationCallback.hal
+++ /dev/null
@@ -1,5 +0,0 @@
-package vendor.huawei.hardware.biometrics.fingerprint@2.1;
-
-interface IFidoAuthenticationCallback {
-	oneway onUserVerificationResult(int32_t result, int64_t opId, vec<int8_t> userId, vec<int8_t> encapsulatedResult);
-};
-- 
2.25.1

